src/app/pages/Admin.tsx

```diff
@@ -22,6 +22,7 @@ export const Admin = () => {
 
 	const [selectedRobot, setSelectedRobot] = useState<string>()
 	const [blockScroll, allowScroll] = useScrollBlock()
+	const [moveRobotsUnlocked, setMoveRobotsUnlocked] = useState<boolean>(false)
```

Comment:
Should be derived from the GameEngine state.

Reply:
Should be fixed